okhttp/src/main/java/okhttp3/internal/Util.java

```diff
@@ -256,23 +256,6 @@ public static String hostHeader(HttpUrl url, boolean includeDefaultPort) {
         : host;
   }
 
-  /** Returns {@code s} with control characters and non-ASCII characters replaced with '?'. */
-  public static String toHumanReadableAscii(String s) {
```

Comment:
gonna hurt anybody who used `.internal`, but it saves a method in OkHttp.

Reply:
Nice.

Reply:
@NightlyNexus 
> gonna hurt anybody who used `.internal`, but it saves a method in OkHttp.

Because these changes are breaking SemVer, why did you include them in the latest release `3.10.0`? Shouldn't have been included in a major release?

We're one of those affected people 😬

cc @swankjesse

Reply:
The classes in `internal` are internal-only implementation details. Please don't depend on them; they _always_ change between releases.